include/drivers/pwm.h

```diff
@@ -55,11 +76,15 @@ typedef int (*pwm_get_cycles_per_sec_t)(const struct device *dev,
 /** @brief PWM driver API definition. */
 __subsystem struct pwm_driver_api {
 	pwm_pin_set_t pin_set;
+#ifdef CONFIG_PWM_CAPTURE_API
+	pwm_pin_capture_t pin_capture;
+	pwm_pin_capture_async_t pin_capture_async;
+#endif /* CONFIG_PWM_CAPTURE_API */
 	pwm_get_cycles_per_sec_t get_cycles_per_sec;
 };
 
 /**
- * @brief Set the period and pulse width for a single PWM output.
+ * @brief Set the period and pulse width for a single PWM input.
```

Comment:
Is this supposed to have changed from output to input?  The rest of the description seems to suggest it's an output.